components/aws/sagemaker/train/src/train.py

```diff
@@ -21,7 +22,7 @@ def create_parser():
   
   parser.add_argument('--job_name', type=str, required=False, help='The name of the training job.', default='')
   parser.add_argument('--role', type=str, required=True, help='The Amazon Resource Name (ARN) that Amazon SageMaker assumes to perform tasks on your behalf.')
-  parser.add_argument('--image', type=str, required=True, help='The registry path of the Docker image that contains the training algorithm.', default='')
+  parser.add_argument('--image', type=str, required=False, help='The registry path of the Docker image that contains the training algorithm.', default='')
```

Comment:
the training image is not required now?

Reply:
As correctly stated in our documentation, it was never meant to be required in the argument parser. The user can choose whether they want:
1. A 1p algorithm - which they specify in the `--algorithm_name` parameter
2. A custom training image - which they specify in the `--image` parameter

Since these are mutually exclusive, it was not actually possible to use a 1p algorithm until I found this issue.

Reply:
Sounds good